manifest.xml

```diff
@@ -91,7 +91,7 @@
       <bt:Urls>
         <bt:Url id="Functions.Script.Url" DefaultValue="https://localhost:3000/dist/functions.js" />
```

Comment:
What about the other two lines? I think you should be able to remove it from all three. functions.html was the main one, and I just kept the other two consistent.

Reply:
Yes, we can remove dist from these too - will push commit

Reply:
I am also going to increase the sleepin runCfTests because I notice that the functions seem to be taking longer to resolve on the VMs, and while the functions actually work, our tests try to capture the value before the function resolves